### PR TITLE
harmony code for waveloading

### PR DESCRIPTION
**Pull request by harmonygeek**
Please review and merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,48 @@
-#WaveLoading
-
-This library provides a *wave loading* animation as a **Drawable**.
+# WaveLoading
+  WaveLoading: This library provides a wave loading animation as a Drawable Element.
 
 ![screenshot](./screenshots/screenshot.gif)
+ 
+## Usage Instructions
```
Comment (sharathks84) [inline README.md:6]: Use HarmonyOS images and avoid using Android images

Comment (harmonygeek) [reply README.md:6]: Removed android images, added an apple image.

**Review by sharathks84 — APPROVED** [review]
OK, Approved

**Review by Gleelakrishna — APPROVED** [review]
ok

**Pull request merged**